### PR TITLE
[DOC-7436] Security audits docs revamp

### PR DESCRIPTION
**Pull request by urbiz-nr**
Prepping new doc structure for https://docs.newrelic.com/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services/, as described in DOC-7436.

This is just a first PR to implement changes in the nav file, add a couple of empty drafts, and verify it works locally.

### CODE REVIEW

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      <th>
+        Services
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+     <tr>
+      <td>
+        HIPAA BAA FAQ
```
Comment (urbiz-nr) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx:48]: Do we have this doc, @jxainguyen?

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      <th>
+        Services
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+     <tr>
+      <td>
+        ISO 27001:2013 Certificate
```
Comment (urbiz-nr) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx:44]: Same, dow e have this doc, @jxainguyen ?

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/soc2.mdx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+      <th>
+        Services
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+     <tr>
+      <td>
+        SOC 2 Type 2
```
Comment (urbiz-nr) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/soc2.mdx:44]: @jxainguyen - same, do we have the doc?

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/soc2.mdx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+      </td> 
+      
+      <td>
+        New Relic One Platform
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        SOC 2 Type 1
```
Comment (urbiz-nr) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/soc2.mdx:62]: @jxainguyen - same, do we have the doc?

Comment (jxainguyen) [reply src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/soc2.mdx:62]: For all of your comments, I've connected with my director & added document links for HIPAA & ISO in a PR that needs to be merged into this. SOC document links weren't provided.

**Comment by jxainguyen** [comment]
Checked with my director and the Gatsby build preview looks great 👍 https://build-97fe9711-4613-4b94-b632-f964eed687ff.gtsb.io/

Unsure why but this time the Snyk tests failed. Once these are able to be resolved, please push to live/prod as the content/structure looks great. Please let me know if I can do anything to assist. Thanks

**Comment by jxainguyen** [comment]
All checks have passed and the build looks great! We'll wait for Anna's final confirm then feel free to publish to live.

Thanks everyone, especially @bradleycamacho and @urbiz-nr for the help troubleshooting & templates - really appreciated!

**Review by urbiz-nr — COMMENTED** [review]
Everything looks good, and tried locally!

**Review by x8a — APPROVED** [review]
Looking good!

**Pull request merged**